pytest_odoo.py

```diff
@@ -131,103 +141,16 @@ def enable_odoo_test_flag():
     odoo.tools.config['test_enable'] = False
 
 
-# Original code of xmo-odoo:
```

Comment:
no need for this ugly hack anymore? :exploding_head:

Reply:
no this PR do not needs it but as mentioned this PR do not support python class created and used in a same python mobule while he does'nt respect python namespace and rely on Odoo addons_path.

In fact the metaclass monkey patch present on this PR was used to this purpose to hijack class namespace while importing model class. That's why it doesn't works with normal python class !

Unfortunately (and hopefully) we can't monkey patch `type` built-in :sweat_smile: